fiftyone/core/models.py

```diff
@@ -781,7 +790,9 @@ def compute_embeddings(
             **kwargs,
         )
 
-    if _is_ultralytics_model(model):
+    if _is_transformers_model(model):
```

Comment:
I went ahead and added this conversion call to `compute_embeddings()` and `compute_patch_embeddings()` so that if our HF model wrappers expose embeddings then these methods will "just work"

Reply:
Awesome! There are going to be a few subtleties if we want to expose this functionality — which I think we should.

- Currently the conversion requires that the model is a detection of classification model, but for embeddings this is not necessary. [Here](https://huggingface.co/docs/transformers/model_doc/deit#transformers.DeiTModel)'s an example where `DeiTModel` can generate the embeddings, and doesn't need the `ForObjectDetection` etc.
- If we are exposing for embeddings, does this mean we are exposing for `compute_similarity()` as well?

Reply:
Looks like `ultralytics.YOLO` models don't support extracting embeddings so I'm going to remove them from this.

Reply:
It's okay to leave this code in here. Maybe we add support for additional Ultralytics models that do expose embeddings in the future. 

As written, if you pass an `ultralytics.YOLO` model to `compute_embeddings()` it will convert it but then you'll get an error saying that the model doesn't expose embeddings, which is exactly the error we want.

Reply:
Any `fom.Model` that implements `fom.EmbeddingsMixin` will automatically become available for use with `compute_similarity()`, yes.

Reply:
Fair enough!